L/LibTracyClient/LibTracyClient@0.9.0/bundled/patches/libTracyClient-freebsd-elfw.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/public/TracyClient.cpp b/public/TracyClient.cpp
```

Comment:
Should this patch be upstreamed?

Reply:
Yep, I'll work to get it upstreamed soon